docs/index.asciidoc

```diff
@@ -357,7 +357,35 @@ Specify the truststore type here. One of `JKS` or `PKCS12`. Default is `JKS`
   * There is no default value for this setting.
 
 A Hash of urls in this format : `"name" => "url"`.
-The name and the url will be passed in the outputed event
+The name and the url will be passed in the outputted event.
+
+The values in urls can be either:
+
+* a string url (which will be issued as an HTTP GET).
+* a sub-hash containing many useful keys provided by the Manticore backend:
+** url: the String url
+** method: (optional) the HTTP method to use (defaults to GET)
+** user: (optional) the HTTP Basic Auth user
+** password: (optional) the HTTP Basic Auth password. The password 
+must be under an auth sub-hash for Manticore, but this plugin accepts it either way.
+** headers: a hash containing key-value pairs of headers. The header must be under 
+an auth sub-hash for Manticore, but this plugin accepts it either way.
+** body: a string (supported only on POST and PUT requests)
+** possibly other options mentioned in the 
+(https://www.rubydoc.info/github/cheald/manticore/Manticore/Client#http-instance_method)[Manticore docs].
+Note that these options are not thoroughly tested and therefore liable to
+break in unexpected ways if we replace the backend.
```

Comment:
I think indentation matters here to keep the text a part of the ASCIIdoc bullet.

I also reworded it a bit to _try_ to indicate that we'll continue to support the arguments that we document above, and that folks are really only on their own if they pass _other_ manticore options.

```suggestion
** possibly other options mentioned in the 
   (https://www.rubydoc.info/github/cheald/manticore/Manticore/Client#http-instance_method)[Manticore docs].
   Note that Manticore options that are not explicitly documented above are not
   thoroughly tested and therefore liable to break in unexpected ways if we
   replace the backend.
```

Reply:
FYI, asciidoc renders the same with and without the formatting change. There's no return before the note starts, so the note is appended to the bullet item.